tests/test_diffusers.py

```diff
@@ -619,39 +621,34 @@ def test_no_generation_regression(self):
             use_hpu_graphs=True,
             gaudi_config=GaudiConfig(use_torch_autocast=False),
         )
-        set_seed(27)
+
+        prompt = "An image of a squirrel in Picasso style"
+        generator = torch.manual_seed(seed)
         outputs = pipeline(
-            prompt="An image of a squirrel in Picasso style",
+            prompt=prompt,
+            generator=generator,
             output_type="np",
         )
 
         if os.environ.get("GAUDI2_CI", "0") == "1":
-            expected_slice = np.array(
-                [
-                    0.68306947,
-                    0.6812112,
-                    0.67309505,
-                    0.70057267,
-                    0.6582885,
-                    0.6325019,
-                    0.6708976,
-                    0.6226433,
-                    0.58038336,
-                ]
-            )
+            target_score = 30.2968
         else:
-            expected_slice = np.array(
-                [0.70760196, 0.7136303, 0.7000798, 0.714934, 0.6776865, 0.6800843, 0.6923707, 0.6653969, 0.6408076]
-            )
+            target_score = 36.774
```

Comment:
Bottomline is chosen as 95% of the mean of the CLIP scores I got from multiple runs (5-10 runs).